client/src/main/java/org/asynchttpclient/DefaultAsyncHttpClient.java

```diff
@@ -255,6 +257,13 @@ public EventLoopGroup getEventLoopGroup() {
         return channelManager.getEventLoopGroup();
     }
 
+    public ClientStats getClientStats() {
+        final ChannelManager channelManager = requestSender.getChannelManager();
+        final long activeConnectionCount = channelManager.getOpenChannels().stream().filter(channel -> !channel.isOpen()).count();
+        final long idleConnectionCount = channelManager.getOpenChannels().stream().filter(Channel::isOpen).count();
```

Comment:
I really wonder if it's possible to get `idleConnectionCount` different from zero.